Assets/AppCenter/Plugins/AppCenterSDK/Distribute/Shared/UpdateAction.cs

```diff
@@ -11,8 +11,8 @@ public enum UpdateAction
         /// <summary>
         /// Action to trigger the download of the release.
         /// </summary>
-        Update,
-
+        Update,

```

Comment:
I am curious, what is the end-of-line change here? If there's an inconsistency between CRLF (Windows) and LF (macOS), yes we should fix it.

![image](https://user-images.githubusercontent.com/4079255/68443237-34215080-0188-11ea-8081-45798544e30c.png)

Reply:
Yes, there was an inconsistency between CRLF (Windows) and LF (macOS). Unity shows a warning about it.